flycheck.el

```diff
@@ -5071,7 +5084,12 @@ CHECKER."
             (list (flycheck-verification-result-new
                    :label "configuration file"
                    :message (if path (format "Found at %S" path) "Not found")
-                   :face (if path 'success 'warning))))))))
+                   :face (if path 'success 'warning)))))
+      ,@(when (not (flycheck-temp-files-writable-p checker))
+          (list (flycheck-verification-result-new
+                 :label "directory"
+                 :message "is not writable"
+                 :face 'error))))))
```

Comment:
Maybe `temp directory` rather than just directory, and `not writable (<path>)`?

Reply:
I'll try that yeah.